src/pages/Account/AdminEvent.tsx

```diff
@@ -0,0 +1,184 @@
+      collective_price: 0,
+    };
+    await postEvent(venueId, event);
+    onHide();
+  };
+  return (
+    <Modal show={show} onHide={onHide}>
+      <div className="form-container">
+        <h2>Create an event</h2>
+        <form onSubmit={handleSubmit(onSubmit)} className="form">
+          <div className="input-group">
+            <label htmlFor="name">Name</label>
+            <input
+              id="name"
+              name="name"
+              className="input-block input-centered"
+              placeholder="Name"
+              ref={register}
+            />
+            {errors.name && (
+              <span className="input-error">{errors.name.message}</span>
+            )}
+          </div>
+          <div className="input-group">
+            <label htmlFor="description">Description</label>
+            <textarea
+              name="description"
+              className="input-block input-centered"
+              placeholder="Description"
+              ref={register}
+            />
+            {errors.description && (
+              <span className="input-error">{errors.description.message}</span>
+            )}
+          </div>
+          <div className="input-group">
+            <label>Start Time</label>
+            <input
+              type="date"
+              min={dayjs().format("YYYY-MM-DD")}
+              name="start_date"
+              className="input-block input-centered"
+              ref={register}
+            />
+            {errors.start_date && (
+              <span className="input-error">{errors.start_date.message}</span>
+            )}
+            <input
+              type="time"
+              name="start_time"
+              className="input-block input-centered"
+              ref={register}
+            />
+            {errors.start_time && (
+              <span className="input-error">{errors.start_time.message}</span>
+            )}
+          </div>
+          <div className="input-group">
+            <label>End Time</label>
+            <input
+              type="date"
+              min={dayjs().format("YYYY-MM-DD")}
+              name="end_date"
+              className="input-block input-centered"
+              ref={register}
+            />
+            {errors.end_date && (
+              <span className="input-error">{errors.end_date.message}</span>
+            )}
+            <input
+              type="time"
+              name="end_time"
+              className="input-block input-centered"
+              placeholder="Time"
+              ref={register}
+            />
+            {errors.end_time && (
+              <span className="input-error">{errors.end_time.message}</span>
+            )}
+          </div>
+          <div className="input-group">
+            <label htmlFor="price">Price</label>
+            <input
+              id="price"
+              name="price"
+              className="input-block input-centered"
+              placeholder="£20"
```

Comment:
add to settings.ts

Reply:
This is very local to here. Why add it to settings?